### DISCUSSION TITLE
ZepetoScriptableObject 에서 함수호출 문제.

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#1213] [5. Q&A]
현재 SDK 1.5.1을 사용중입니다.

ZepetoScriptableObject 로 스크립트를 만들고, 함수를 만들어 호출을 하면, 호출하려는 함수가 함수가 아니라는 에러가 납니다.
스크립트애이블오브젝트에선 함수를 정의해서 사용할수 없는건가요?

//ZepetoScriptableObject Script
export default class ImageCollection{
    public imageArray:Sprite[];
    public Test()
    {
        console.log("TEST");
    }
}

//Main Script
    public obj:ZepetoScriptableObject<ImageCollection>;
    Start() {    
        console.log("Hello ZEPETO Script");
        this.obj.targetObject.Test();
    }

//Error Log
System.Exception: file:///...../SampleScript.js:7: TypeError: this.obj.targetObject.Test is not a function

확인부탁드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 내부에서 확인 후 말씀 드리겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 확인 결과 현재 스펙에서는 ZepetoScriptableObject 의 할당 된 TypeScript 에는 함수 사용을 할 수 없습니다.
추후 지원 여부는 검토 중이나 확답이 어려운 점 양해 부탁 드립니다.
감사합니다.
